Bamboo/backend/bamboo-backend/src/services/user/user.service.ts

```diff
@@ -49,4 +50,41 @@ export class UserService {
     return existingUser
   
   }
+
+  async initialiseAccount(accountType: string, jwt: JwtPayload){
```

Comment:
initialiseAccount should not contain logic to create the `Student`/`Tutor` record in the DB -- this should be done in a separate service; one for student (e.g. studentService) and one for tutor (e.g. tutorService)